release/_releases_json.sh

```diff
@@ -31,6 +37,28 @@ function _merge_json_snippets {
 	fi
 }
 
+function _process_new_product {
+	local product_name="${1}"
+
+	local downloaded_releases_json="${_RELEASE_ROOT_DIR}/release-data/promotion/files/0000-00-00-downloaded_releases.json"
+
+	if [ ! -f "${downloaded_releases_json}" ]
+	then
+		lc_log INFO "Downloading https://releases.liferay.com/releases.json to ${downloaded_releases_json}."
+
+		lc_download https://releases.liferay.com/releases.json "${downloaded_releases_json}"
+	fi
+
+	if grep "${LIFERAY_RELEASE_VERSION}" "${downloaded_releases_json}"
+	then
+		lc_log INFO "The version ${LIFERAY_RELEASE_VERSION} is already in releases.json"
+
+		return
```

Comment:
@Wykenz, shouldn't we use `return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"`?

Actually, if we use `return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"` won't it still execute the functions `_promote_product_versions dxp, _promote_product_versions portal, _merge_json_snippets`?

Is the execution of these functions really necessary if there is no need to update the file `releases.json`?

I believe you may need something like this [code](https://github.com/liferay/liferay-docker/blob/2c5a878d4b608b0a569e80a85abbb2df05c6531e/release/release_gold.sh#L180-L185) :slightly_smiling_face: